src/vdxf/index.ts

```diff
@@ -281,4 +283,10 @@ export class VerusIDSignature extends VDXFObject {
       serializekey: this.serializekey
     };
   }
+}
+
+export class PNGImageVdxfObject extends BufferDataVdxfObject {
```

Comment:
If this extended HexDataVdxfObject I think you wouldn't even need the constructor line